junebug/channel.py

```diff
@@ -259,6 +259,7 @@ def _application_config(self):
             'redis_manager': self.config.redis,
             'inbound_ttl': self.config.inbound_message_ttl,
             'outbound_ttl': self.config.outbound_message_ttl,
+            'message_rate_bucket': self.config.inbound_message_rate_bucket,
```

Comment:
`window` is great, that's the word I was looking for. The reason the `inbound` is missing here is because it's for the message forwarding worker, which only deals with inbound messages, happy to change it though.